std_msgs/Bool mode
std_msgs/Float32 velocity
std_msgs/Float32 heading
geometry_msgs/Point position